Scenes/EncountersMenuScene.gd

```diff
@@ -320,7 +320,12 @@ func _react(_action: String, _args):
 		
 		setState("speciesmenu")
 		return
-	
+		
+	if(_action == "closenpclist"):
+		setState("")
+		GM.main.playAnimation(StageScene.Solo, "stand")
+		return
```

Comment:
You were right this implementation is better. Mine worked well until I added ability to meet characters. Wasn't really a big deal but since I had your version to switch I chose to do so.